specification/datastore/DatastoreApis.ts

```diff
@@ -56,7 +56,7 @@ export const DatastoreApiSchemas = {
       adminSignature: signatureValidation
         .optional()
         .describe('A signature from the admin Identity'),
-      adminFunction: z.object({
+      adminRunner: z.object({
         ownerType: z
           .enum(['table', 'crawler', 'function', 'datastore'])
```

Comment:
this line should changed function to runner

Reply:
Fixed